docs/nodes/nodes-library/trigger-nodes/GitlabTrigger/README.md

```diff
@@ -0,0 +1,30 @@
+---
+permalink: /nodes/n8n-nodes-base.gitlabTrigger
+---
+
+# GitLab Trigger
+
+[GitLab](https://gitlab.com/) is a web-based DevOps lifecycle tool that provides a Git-repository manager providing wiki, issue-tracking and continuous integration/continuous deployment pipeline features, using an open-source license, developed by GitLab Inc.
```

Comment:
Hey, can you make sure that this description here is same as in the GitLab node docs? Also, the directory should be named as 'GitLabTrigger'.